wiki/misconceptions.md

```diff
@@ -0,0 +1,187 @@
+# Misconceptions
+
+{title candidates: Misconceptions, Rebuttals, Common criticism}
+
+Misconceptions and common arguments against on-chain governance, coin voting and other aspects of Decred system.
+
+## Coin voting is plutocracy
+ 
+Counterarguments:
+
+- Company shareholder governance is also plutocracy, but nobody complains about that.
+- Voluntary: "plutocracy" is often used to mean non-voluntary (forced) participation, e.g. when you are born into a such society. You were not asked if you would like to participate. In public blockchains, on the opposite, no plutocracy happens to you until you voluntary join a given system.
```

Comment:
As far as I can tell "plutocracy" has historically really only been used to refer to the dominant power in a particular geographical location, the monarchy/empire/state. The point about it not usually being applied to opt-in systems is a good one, it doesn't make a lot of sense that people would opt in to something that is characterized by the rich enriching themselves, given reasonable alternatives.